checks/check_status.py

```diff
@@ -127,7 +127,8 @@ def validate_api_key(config):
         if proxy:
             request_proxy = {'https': "http://{user}:{password}@{host}:{port}".format(**proxy)}
         r = requests.get("%s/api/v1/validate" % config['dd_url'].rstrip('/'),
-            params={'api_key': config.get('api_key')}, proxies=request_proxy, timeout=3)
+            params={'api_key': config.get('api_key')}, proxies=request_proxy, timeout=3,
+                         verify=(not config['skip_ssl_validation']))
```

Comment:
Also even if all is apparently well, we're not 100% comfortable with saying "API Key is valid" when we cannot guarantee the connection hasn't been compromised. So if we could tweak the message [here line 144](https://github.com/DataDog/dd-agent/pull/3346/files#diff-74e3374e663661c0fc395cfe46d645d4R144) to say something along these lines (when SSL/TLS validation is disabled): `"API key appears valid. However, SSL validation has been disabled - please beware."` , I think would maybe be preferable.